index.coffee

```diff
@@ -43,20 +40,20 @@ router.get '/', (ctx)->
   """
 
 router.get  '/week', week = (ctx)->
-  events = await this_week()
+  events = await Event.this_week()
   ctx.assert events.length, 404, 'No events this week'
   ctx.body = slack_payload events, 'Events this week'
 
 router.post '/week', (ctx)->
   await post_to_slack await week ctx
 
 router.get  '/today', today = (ctx)->
-  events = await today()
+  events = await Event.today()
```

Comment:
This `today` function was colliding with the imported `Event.today` method, so we fixed the import statement.

Reply:
this one is a good substitution, nice catch!